plugin-mainmenu/xdgcachedmenu.cpp

```diff
@@ -73,6 +73,12 @@ XdgCachedMenu::XdgCachedMenu(MenuCache* menuCache, QWidget* parent): QMenu(paren
 {
     // qDebug() << "CREATE MENU FROM CACHE" << menuCache;
     MenuCacheDir* dir = menu_cache_get_root_dir(menuCache);
+
+    // get current desktop name or fallback to LXQt
+    const QByteArray xdgDesktop = qgetenv("XDG_CURRENT_DESKTOP");
+    const QByteArray desktop = xdgDesktop.isEmpty() ? "LXQt:X-LXQt:LXQT:X-LXQT" : xdgDesktop;
```

Comment:
I've thought about it, and I'm not sure. See [how](https://github.com/lxde/lxqt-panel/blob/master/plugin-mainmenu/lxqtmainmenu.cpp#L220) the libqtxdg-based version sets it. We set `XDG_CURRENT_DESKTOP` to `LXQt`, but the desktop files might set it to `X-LXQt` according to the spec. But what about the 't'? Capital or not? I don't know.